steps/magic-pod/0.54.1/step.yml

```diff
@@ -0,0 +1,111 @@
+title: Magic Pod
+summary: |
+  Magic Pod UI test step
+description: |
+  You can execute E2E testing of iOS/Android apps on Magic Pod (https://magic-pod.com).
+website: https://github.com/magic-Pod/bitrise-step-magicpod-uitest
+source_code_url: https://github.com/magic-Pod/bitrise-step-magicpod-uitest
+support_url: https://github.com/magic-Pod/bitrise-step-magicpod-uitest/issues
+published_at: 2020-06-02T22:24:46.286263+09:00
+source:
+  git: https://github.com/Magic-Pod/bitrise-step-magic-pod.git
+  commit: 3503680420460bbd2ab75446c2f6da2767e19df3
+project_type_tags:
+- ios
+- android
```

Comment:
Do you support multi platform apps like flutter and react-native? Not mandatory but could recommend to more users if such.

Reply:
Thank you for recommendation.  I think our service can handle such multi platform apps, but not especially customized for them.  We will think of such tags in the next version.